pkg/database/audit_entry.go

```diff
@@ -75,3 +77,28 @@ func (db *Database) PurgeAuditEntries(maxAge time.Duration) (int64, error) {
 		Delete(&AuditEntry{})
 	return result.RowsAffected, result.Error
 }
+
+// ListAudits returns the list audit events which match the given criteria.
+// Warning: This list may be large. Use Realm.Audits() to get users scoped to a realm.
+func (db *Database) ListAudits(p *pagination.PageParams, scopes ...Scope) ([]*AuditEntry, *pagination.Paginator, error) {
```

Comment:
Something we should do (eventually): `realm.ListAudits` should call `db.ListAudits` with an additional scope that limits it to the realm id. And then all the other realm methods + db methods.

Reply:
@whaught ^